File: common/src/main/java/org/bread_experts_group/breadlib/test/TestBlockEntity.java
package org.bread_experts_group.breadlib.test;

import net.minecraft.core.BlockPos;
import net.minecraft.world.level.block.entity.BlockEntity;
import net.minecraft.world.level.block.state.BlockState;

public class TestBlockEntity extends BlockEntity {
    public TestBlockEntity(BlockPos pPos, BlockState pBlockState) {
        super(BlockEntityTypeTest.TEST_TYPE.get(), pPos, pBlockState);
    }
}

File: common/src/main/kotlin/org/bread_experts_group/breadlib/test/TestBlockEntity.kt
package org.bread_experts_group.breadlib.test

import net.minecraft.core.BlockPos
import net.minecraft.world.level.block.state.BlockState
import org.bread_experts_group.breadlib.BreadLib
import org.bread_experts_group.breadlib.extensions.block.BreadLibBlockEntity

class TestBlockEntity(
	pos: BlockPos,
	blockState: BlockState
) : BreadLibBlockEntity(pos, blockState, BreadLib.MOD_ID)